tests/queries/0_stateless/03405_ssd_cache_incorrect_min_max_lifetimes_and_block_size.sql

```diff
@@ -0,0 +1,18 @@
+-- Tags: no-ordinary-database
+
+-- Tests that various conditions are checked during creation of a ssd_cache layout of a dictionary
+
+-- Github issue #78314
+
+DROP DICTIONARY IF EXISTS dx;
+
+SELECT 'BLOCK_SIZE is a negative value.';
+CREATE DICTIONARY dx (col Int64 default null) PRIMARY KEY (col) SOURCE(NULL()) LAYOUT(SSD_CACHE(BLOCK_SIZE -1)) LIFETIME(1); -- { serverError 36 }
```

Comment:
`36` will work but the cleartext error name from src/Common/ErrorCodes.cpp would also work:

```cpp
-- { serverError BAD_ARGUMENTS }
```

Reply:
a833c89a8e6cae4b1a2066122217f036d9361f30

Reply:
Cool, thanks. No need to post commit SHAs (I wouldn't usually check the individual commits anyways - I just look at the entire PR) 😄 Rather, feel free to mark the comment as resolved.

Reply:
kinda old habits 😄